mayastor/src/bdev/dev/nvmf.rs

```diff
@@ -192,6 +192,19 @@ impl CreateDestroy for Nvmf {
                 name: self.name.clone(),
             })?;
 
+        if bdev_count == 0 {
+            error!("No nvme bdev created, no namespaces?");
+            // Remove partially created nvme bdev which doesn't show up in
+            // the list of bdevs
+            let errno = unsafe { bdev_nvme_delete(cname.as_ptr()) };
+            info!(
+                "removed partially created {}, returned {}",
```

Comment:
Should the message say "removed partially created **bdev** {} ...."?

Reply:
I guess for accuracy, it should. Fixed.